packages/shared/lib/core/governance/utils/isVotingForProposal.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from 'svelte/store'
+
+import { TrackedParticipationOverview } from '@iota/wallet'
+
+import { getParticipationOverview } from '@core/account/api'
+import { selectedAccount } from '@core/account/stores'
+
+import { selectedProposal } from '../stores'
+
+export async function isVotingForProposal(): Promise<boolean> {
+    const overview = await getParticipationOverview(get(selectedAccount)?.index ?? 0)
```

Comment:
I don't think this should default to 0. This would result in the unexpected behavior that if the currently selected account doesn't have an index, it would get the wrong overview.